### ISSUE TITLE
Feature request - abbreviations

### ISSUE THREAD

**Issue by cdbdev** [notepad-plus-plus/notepad-plus-plus#9540]
Hi,

Could be this was asked before, but I did not find any comment about it. So sorry in advance :)

Is it possible to add some sort of 'abbreviations' functionality to notepad++ that works similar to the functionality in SciTE editor?

Thx

**Comment by sasumner** [comment]
Aren't there some plugins for this?  
FingerText, Snippets are a couple that come to mind...

**Comment by cdbdev** [comment]
Plugins for abbreviations? Snippets indeed, but abbreviations work a little bit differently. You type in a string and it is replaced by an expansion.

Also, can something like this or even snippets be included in the standard?

**Comment by cdbdev** [comment]
Ok just read about Fingertext and it seems like the same functionality. But I've read it is still in Alpha status... Would be nice if it was included by default

**Comment by sasumner** [comment]
Of course, there's always "auto-completion" in the standard product, but that's a tad-bit different.

Typically functionality in plugins isn't brought into the standard product, mainly because the limited number of developers don't want to expend time recreating something that exists, albeit in a different manner of deployment.

**Comment by ArkadiuszMichalski** [comment]
You can try Emmet (aka old Zen Coding) and define your own abbreviation.
